content/webapp/tsconfig.server.json

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "module": "commonjs",
+    "outDir": "dist",
+    "noEmit": false
```

Comment:
Why do we want to emit?

Reply:
Good call - I've simplified it to the only reason we need a separate one, being the module type.